deploy/helm/kubecf/templates/sizing.yaml

```diff
@@ -13,212 +13,58 @@ metadata:
+    {{- $instance_groups = append $instance_groups "asactors" }}
+    {{- $instance_groups = append $instance_groups "asmetrics" }}
+    {{- $instance_groups = append $instance_groups "asnozzle" }}
+    {{- end }}{{/* if autoscaler */}}
+
+    {{- /* CredHub */}}
+    {{- if .Values.features.credhub.enabled }}
+    {{- $instance_groups = append $instance_groups "credhub" }}
+    {{- end }}{{/* if credhub */}}
+
+    {{- /* Instances groups where existance depends on whether Eirini is enabled */}}
+    {{- if .Values.features.eirini.enabled }}
+    {{- $instance_groups = append $instance_groups "bits" }}
+    {{- $instance_groups = append $instance_groups "eirini" }}
+    {{- else }}
+    {{- $instance_groups = append $instance_groups "diego-cell" }}
+    {{- end }}{{/* if eirini */}}
 
     # Temporarily fixes https://github.com/SUSE/kubecf/issues/241.
     # TODO: properly investigate and fix log-cache.
     - type: replace
       path: /instance_groups/name=doppler/instances
       value: 1
 
-{{- if .Values.sizing.log_api.instances }}
-    - type: replace
-      path: /instance_groups/name=log-api/instances
-      value: {{.Values.sizing.log_api.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=log-api/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.nats.instances }}
-    - type: replace
-      path: /instance_groups/name=nats/instances
-      value: {{.Values.sizing.nats.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=nats/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.router.instances }}
-    - type: replace
-      path: /instance_groups/name=router/instances
-      value: {{.Values.sizing.router.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=router/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.routing_api.instances }}
-    - type: replace
-      path: /instance_groups/name=routing-api/instances
-      value: {{.Values.sizing.routing_api.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=routing-api/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.scheduler.instances }}
-    - type: replace
-      path: /instance_groups/name=scheduler/instances
-      value: {{.Values.sizing.scheduler.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=scheduler/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.uaa.instances }}
-    - type: replace
-      path: /instance_groups/name=uaa/instances
-      value: {{.Values.sizing.uaa.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=uaa/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.tcp_router.instances }}
+{{- $root := . }}
```

Comment:
Instead of `$root := .`, we can access the root with `$.Release.Name` for example.

Reply:
Changed in 90c8a63399c4c1c0082a58ba14ddaa4f72d71b3a.